src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java

```diff
@@ -260,75 +262,96 @@ private static void addMultiGraphAggregation(TravelOptions travelOptions, JSONOb
                 travelOptions.getMultiGraphAggregationMaxResultValueRatio(),
                 travelOptions.getMultiGraphAggregationFilterValuesForSourceOrigins()).anyMatch(Objects::nonNull)) {
             JSONObject multigraphAggregation = new JSONObject();
-            AggregationConfig aggregationConfig = buildAggregationConfigFromTravelOptions(travelOptions);
-            fillJsonAggregationConfig(aggregationConfig, multigraphAggregation);
+            AggregationConfiguration aggregationConfiguration = buildAggregationConfigFromTravelOptions(travelOptions);
+            fillJsonAggregationConfig(aggregationConfiguration, multigraphAggregation);
             multiGraph.put(Constants.MULTIGRAPH_AGGREGATION, multigraphAggregation);
         }
     }
 
-    private static AggregationConfig buildAggregationConfigFromTravelOptions(TravelOptions travelOptions) {
-        return AggregationConfig.builder().ignoreOutlier(travelOptions.getMultiGraphAggregationIgnoreOutlier())
-                .maxResultValue(travelOptions.getMultiGraphAggregationMaxResultValue()).maxResultValueRatio(travelOptions.getMultiGraphAggregationMaxResultValueRatio())
-                .minSourcesCount(travelOptions.getMultiGraphAggregationMinSourcesCount()).minSourcesRatio(travelOptions.getMultiGraphAggregationMinSourcesRatio())
-                .outlierPenalty(travelOptions.getMultiGraphAggregationOutlierPenalty()).type(travelOptions.getMultiGraphAggregationType())
-                .sourceParameters(travelOptions.getMultiGraphSourceParameters()).filterValuesForSourceOrigins(travelOptions.getMultiGraphAggregationFilterValuesForSourceOrigins()).build();
+    private static AggregationConfiguration buildAggregationConfigFromTravelOptions(TravelOptions travelOptions) {
+        return new AggregationConfiguration.AggregationConfigurationBuilder()
+                .ignoreOutlier(travelOptions.getMultiGraphAggregationIgnoreOutlier())
+                .maxResultValue(travelOptions.getMultiGraphAggregationMaxResultValue())
+                .maxResultValueRatio(travelOptions.getMultiGraphAggregationMaxResultValueRatio())
+                .minSourcesCount(travelOptions.getMultiGraphAggregationMinSourcesCount())
+                .minSourcesRatio(travelOptions.getMultiGraphAggregationMinSourcesRatio())
+                .outlierPenalty(travelOptions.getMultiGraphAggregationOutlierPenalty())
+                .type(travelOptions.getMultiGraphAggregationType())
+                .aggregationInputParameters(travelOptions.getMultiGraphAggregationInputParameters())
+                .filterValuesForSourceOrigins(travelOptions.getMultiGraphAggregationFilterValuesForSourceOrigins())
```

Comment:
it seems postAggregationFactor and gravitationExponent are missing here?

also please check if they are maybe missing in other locations